circle.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           name: Selenium Start
           background: true
           command: |
-            ./node_modules/webdriver-manager/bin/webdriver-manager update
+            ./bin/webdriver-manager update
```

Comment:
This was to just call the correct webdriver-manager in circle. We need these binaries for some tests.